pkg/eestream/encode.go

```diff
@@ -267,8 +267,8 @@ func (er *EncodedRanger) Range(ctx context.Context, offset, length int64) ([]io.
 // CalcPieceSize calculates what would be the piece size of the encoded data
 // after erasure coding data with dataSize using the given ErasureScheme.
 func CalcPieceSize(dataSize int64, scheme ErasureScheme) int64 {
-	stripes := dataSize / int64(scheme.StripeSize())
-	if dataSize%int64(scheme.StripeSize()) != 0 {
+	stripes := (dataSize + uint32Size) / int64(scheme.StripeSize())
```

Comment:
why do you have to add the uint32Size? just curious

Reply:
this is actually from kaloyan's PR here https://github.com/storj/storj/pull/1525